### PR TITLE
Update changelog for 0.3.1

### PR DESCRIPTION
**Pull request by parente**

### CODE REVIEW

**Comment by parente** [comment]
@ericdill @danielballan unless you guys object, I'd like to cut a 0.3.1 release with the two minor fixes mentioned in the changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## v0.3.1
+
+* [#26](https://github.com/danielballan/nbexamples/pull/26) Added sort order to the notebooks in the Exampels tab: title (if set) else filename
```
Comment (ericdill) [inline CHANGELOG.md]: s/Exampels/Examples

**Review by ericdill — CHANGES_REQUESTED** [review]

**Review by ericdill — APPROVED** [review]

**Comment by ericdill** [comment]
LGTM. Thanks @parente

**Comment by danielballan** [comment]
Go for it!

**Pull request merged**